src/pages/ForgotPasswordV2.jsx

```diff
@@ -0,0 +1,144 @@
+import {
+    Box,
+    Button,
+    Center,
+    FormControl,
+    FormLabel,
+    Heading,
+    Input,
+    Link,
+    SimpleGrid,
+    Text,
+    useToast,
+    Image,
+    VStack,
+  } from '@chakra-ui/react';
+  import { yupResolver } from '@hookform/resolvers/yup';
+  import { useForm } from 'react-hook-form';
+  import { useNavigate } from 'react-router-dom';
+  import { useState } from 'react';
+  import * as yup from 'yup';
+  import { sendResetPasswordPrompt } from '../utils/firebaseAuthUtils';
+  import logo from '../Assets/Logo.png'; // Make sure you have this logo or replace it with your own
+  
+  const ForgotPasswordV2 = () => {
+    return (
+      <SimpleGrid columns={2} spacing={0} height={'100vh'}>
+        <Box backgroundColor={'lightgray'}>
+          <Center marginTop={20}>
+            <VStack>
+              <Heading marginTop="30%">Stand Up To Trash</Heading>
+              <Image
+                marginTop="10%"
+                borderRadius="full"
+                boxSize="300px"
+                src={logo}
+                alt="Logo"
+              />
+            </VStack>
+          </Center>
+        </Box>
+        <Box marginTop={40}>
+          <ForgotPasswordForm />
+        </Box>
+      </SimpleGrid>
+    );
+  };
+  
+  const resetSchema = yup.object({
+    email: yup.string().email().required('Please enter your email address'),
+  });
+  
+  const ForgotPasswordForm = () => {
+    const [email, setEmail] = useState('');
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/latest/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'setEmail' is assigned a value but never used.